user/articles/08-reference/02-configuration-via-lcp-json.markdown

```diff
@@ -1,13 +1,11 @@
 ---
-header-id: configuring-via-lcp-json
+header-id: configuration-via-lcp-json
 ---
 
-# Configuring via LCP.json
+# Configuration via LCP.json
 
-DXP Cloud uses the `LCP.json` file to configure your container. You can use this 
-file to configure properties like the service ID, memory, number of CPUs, 
-environment variables, volumes, and much more. This table lists the properties 
-you can add to `LCP.json`: 
+DXP Cloud uses the `LCP.json` file to configure the containers of your services. 
```

Comment:
"to configure the containers of your services. " > "to configure the services of your environments."

I think that's better since users can't configure the container itself, just the service.

Reply:
I actually like this paragraph since it gives an idea of what it can be done with the file without reading the entire table:

"You can use this  file to configure properties like the service ID, memory, number of CPUs,  environment variables, volumes, and much more. This table lists the properties  you can add to `LCP.json`"

What do you think of bringing it back?